src/data/faq_de.json

```diff
@@ -2210,47 +2294,214 @@
+                },
+                {
+                    "term": "GitHub",
+                    "anchor": "github",
+                    "description": "GitHub ist eine Plattform, die es vielen verschiedenen Personen ermöglicht, gleichzeitig an einem Projekt zu arbeiten. Sie kann die Beiträge aller Personen zusammenführen und dokumentiert jegliche Änderungen."
                 },
                 {
                     "term": "Gültigkeit eines EU digitalen COVID-Zertifikats",
                     "anchor": "validity",
-                    "description": "Wir unterscheiden grundsätzlich<ul><li>die Gültigkeit des digitalen Zertifikats (die technische Gültigkeit) und</li><li>die Gültigkeit des Impfschutzes (die fachliche Gültigkeit).</li></ul>Das digitale Zertifikat hat ein Ausstellungsdatum und ein technisches Ablaufdatum (so wie auch Ihr Reisepass und PA). Die Gültigkeit des Zertifikats endet 1 Jahr nach Ausstellung. <br/><br/> <b> Impfzertifikat </b> <br/><br/> In dem Zertifikat werden Daten transportiert, in diesem Falle Informationen zur Impfung. Diese Impfung fand an einem Datum X statt und gilt ab dann (nach aktuellem Stand der wissenschaftlichen Erkenntnisse) z.&nbsp;B. in Deutschland 1 Jahr. Danach ist der Impfschutz erloschen. Die Dauer des Impfschutzes bezeichnen wir als 'fachliche' Gültigkeit. Die fachliche Gültigkeit der Impfung zählt also ab dem letzten Impfdatum (Datum X). <br/><br/> Ein Zertifikat ist solange valide, bis entweder die technische Gültigkeit oder die fachliche Gültigkeit abläuft."
+                    "description": "Wir unterscheiden grundsätzlich<ul><li>die Gültigkeit des digitalen Zertifikats (die technische Gültigkeit) und</li><li>die Gültigkeit des Impfschutzes (die fachliche Gültigkeit).</li></ul>Das digitale Zertifikat hat ein Ausstellungsdatum und ein technisches Ablaufdatum (so wie auch Ihr Reisepass und PA). Die Gültigkeit des Zertifikats endet 1 Jahr nach Ausstellung. <br/><br/> <b> Impfzertifikat </b> <br/><br/> In dem Zertifikat werden Daten transportiert, in diesem Falle Informationen zur Impfung. Diese Impfung fand an einem Datum X statt und gilt ab dann (nach aktuellem Stand der wissenschaftlichen Erkenntnisse) z.&nbsp;B. in Deutschland 1 Jahr. Danach ist der Impfschutz erloschen. Die Dauer des Impfschutzes bezeichnen wir als 'fachliche' Gültigkeit. Die fachliche Gültigkeit der Impfung zählt also ab dem letzten Impfdatum (Datum X). <br/><br/> Ein Zertifikat ist solange valide, bis entweder die <a href='#glossary_technical_validity'>technische Gültigkeit</a> oder die <a href='#glossary_effective_validity'>fachliche Gültigkeit</a> abläuft."
                 }
             ],
             "I": [
+                {
+                    "term": "Impfbescheinigung",

```

Comment:
Impfbescheinigung oder Impfzertifikat?

Reply:
That's definitely a valid concern, this is currently being investigated, thanks!